### PR TITLE
release_management: Add cargo-vendor-filterer option

### PR DESCRIPTION
**Pull request by bgurney-rh**

### CODE REVIEW

File: release_management/create_artifacts.py [hunk 1]
```diff
@@ -111,9 +118,14 @@ def _stratisd_artifacts(namespace):
 
     r_v = ReleaseVersion(release_version, namespace.pre_release_suffix)
 
+    filtered = bool(namespace.vendor_method == "filtered")
```
Comment (mulkieran) [inline release_management/create_artifacts.py:121]: You can omit the ```bool``` call, since the ```==``` expression evaluates to a bool, anyway.

**Review by mulkieran — CHANGES_REQUESTED** [review]
One minor request...and then we can work on next steps.

**Comment by bgurney-rh** [comment]
Rebased and resolved conflicts.

File: release_management/create_artifacts.py (hunk 1 above)
Comment (bgurney-rh) [reply release_management/create_artifacts.py]: Resolved.

**Comment by bgurney-rh** [comment]
Rebased to add removal of `bool()` as a fixup to the create_artifacts.py commit, and to leave the `run cargo vendor-filterer` commit as the last commit.

**Review by mulkieran — CHANGES_REQUESTED** [review]
As far as I can tell, all that's left to do is drop the chunks of code that print whether filtering is being done or not, and documenting the filterer parameter that you add and this is ready-to-merge.

**Comment by bgurney-rh** [comment]
Squashed into a single commit.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**